### ISSUE TITLE
Training loss curves

### ISSUE THREAD

**Issue by bblgbr** [LuizScarlet/StableCodec#4]
Thanks for your good job and the open source! It's really a meaningful job and I totally understand that the choice of not sharing the training scripts. However, could you provide the training loss curves to assist me to reproduce the training?

**Comment by LuizScarlet** [comment]
Hi @bblgbr , thanks for your patience. We’ve added recent training logs for reproducing StableCodec to the repo. In our experiments, we recorded the validation performance on Kodak. You can find the raw logs here:
[StableCodec/results/log_stage1.txt](https://github.com/LuizScarlet/StableCodec/blob/main/results/log_stage1.txt)
[StableCodec/results/log_stage2_ft2.txt](https://github.com/LuizScarlet/StableCodec/blob/main/results/log_stage2_ft2.txt)
[StableCodec/results/log_stage2_ft8.txt](https://github.com/LuizScarlet/StableCodec/blob/main/results/log_stage2_ft8.txt)
Additionally, the evaluation results reported in our paper are now available in:
[StableCodec/results/results.txt](https://github.com/LuizScarlet/StableCodec/blob/main/results/results.txt)
Thanks again for your interest : )

**Comment by bblgbr** [comment]
Thank you very much for your response and for providing detailed training and testing information. We sincerely appreciate your contribution to the compression community — this is an excellent piece of research work.
